src/App.js

```diff
@@ -1,23 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Books from './redux/books/Books';
+import Categories from './redux/categories/Categories';
+import NavBar from './components/NavBar';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <NavBar />
+      <Routes>
+        <Route path="/" element={<Books />} />
```

Comment:
- [ ] **[REQUIRED CHANGES]** Please be sure to add the `exact` to the root path, to avoid your app constantly redirecting you to the main page.
- [ ] **[REQUIRED CHANGES]** The react component (declared as `element`) should be **_nested_** inside of the `Route`. Could you change it for us for this route and the next one?

Reply:
- Since the Books page is the default view as per the instruction its path is '/' as I've written.
- The react component 'element' is nested inside the Route tag, so I do not really understand what you want me to change.

Reply:
@Kingstalux prior to `react-router` v6 we had to invoke `exact` on routed to stop match the `url` with value of `path`, it's okay to skip the requested change since your are using v6+.

Reply:
Okay, thanks